js/poison-ui.js

```diff
@@ -0,0 +1,13 @@
+let PoisonUI = new Component('#poison', {
+  data: {
+    //timer
+    time: effects.poison.counter,
+    //damage to player
+    damage: effects.poison.dmg,
+    //pause flag
+    isPaused: effects.poison.isPaused
+  },
+  template: function() {
```

Comment:
any reason these UIs aren't returning any template html?

Reply:
It doesn't work on the html elements when I return them in the template. So I have the UI pieces in the dom so I can attach the components to something. Nothing needs to be rendered because the animations use another set of html elements.